src/Slices/ServiceInventory/UI/ServiceInventory.test.tsx

```diff
@@ -1,96 +1,40 @@
 import React, { act } from "react";
 import { MemoryRouter, useLocation } from "react-router-dom";
 import { Page } from "@patternfly/react-core";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { fireEvent, render, screen, within } from "@testing-library/react";
 import { userEvent } from "@testing-library/user-event";
 import { StoreProvider } from "easy-peasy";
 import { axe, toHaveNoViolations } from "jest-axe";
-import { Either, RemoteData } from "@/Core";
-import {
-  QueryResolverImpl,
-  ServiceInstancesQueryManager,
-  ServiceInstancesStateHelper,
-  InstanceResourcesQueryManager,
-  InstanceResourcesStateHelper,
-  CommandResolverImpl,
-  DeleteInstanceCommandManager,
-  BaseApiHelper,
-  TriggerSetStateCommandManager,
-  getStoreInstance,
-  TriggerForceStateCommandManager,
-  DestroyInstanceCommandManager,
-} from "@/Data";
-import { defaultAuthContext } from "@/Data/Auth/AuthContext";
+import { HttpResponse, http } from "msw";
+import { setupServer } from "msw/node";
+import { RemoteData } from "@/Core";
+import { getStoreInstance } from "@/Data";
 import {
   Service,
   ServiceInstance,
   Pagination,
   StaticScheduler,
-  DynamicQueryManagerResolverImpl,
-  DynamicCommandManagerResolverImpl,
   MockEnvironmentModifier,
   DeferredApiHelper,
   dependencies,
 } from "@/Test";
 import { words } from "@/UI";
 import { DependencyProvider, EnvironmentHandlerImpl } from "@/UI/Dependency";
 import { ModalProvider } from "@/UI/Root/Components/ModalProvider";
-import { TriggerInstanceUpdateCommandManager } from "@S/EditInstance/Data";
 import { Chart } from "./Components";
 import { ServiceInventory } from "./ServiceInventory";
 
 expect.extend(toHaveNoViolations);
 
 function setup(service = Service.a, pageSize = "") {
+  const client = new QueryClient({
+    defaultOptions: { queries: { retry: false } },
```

Comment:
Is linting not complaining about the format? I've seen this formatted in three different ways so far x)

Reply:
no, it does it on its own, it's linted before merge :D